meshtastic/node_runtime/seturl/context.py

```
@@ -0,0 +1,19 @@
+from dataclasses import dataclass
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from meshtastic.node import Node
+
+
+@dataclass(frozen=True)
+class _SetUrlAdminContext:
+    """Admin-channel write path metadata resolved before planning/execution."""
+
+    admin_write_node: "Node"
```

Ruff (0.15.7)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)